scripts/zipformer-transducer/streaming_v1/qnn/run-2023-02-20.sh

```
@@ -0,0 +1,58 @@
+
+chunk_size=32
+
+if [ $# -ge 1 ]; then
+  chunk_size="$1"
+fi
+
+echo "chunk_size: $chunk_size"
+
+echo "download models"
+git clone https://huggingface.co/pfluo/k2fsa-zipformer-chinese-english-mixed
+pushd k2fsa-zipformer-chinese-english-mixed/exp
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)